### DISCUSSION TITLE
✅ VisionOS support

### DISCUSSION THREAD

**Discussion by tshazar** [isontheline/pro.webssh.net#1072] [Ideas]
Basic visionOS support, at the least in compatibility mode, would be wonderful. 
There should not be many adjustments needed. Is this already on a build timeline?

[developer.apple.com]https://developer.apple.com/documentation/visionos/bringing-your-app-to-visionos)

**Comment by isontheline** [comment]
@tshazar visionOS support is on its way :
<img width="927" alt="image" src="https://github.com/isontheline/pro.webssh.net/assets/44212923/50be820c-ec49-4bd1-afef-180045f2203e">

I need to fix some parts of the CI-CD :
<img width="237" alt="image" src="https://github.com/isontheline/pro.webssh.net/assets/44212923/fb9192f4-d0fc-4920-bd33-ff5da272f84f">

And some crashes :
<img width="725" alt="image" src="https://github.com/isontheline/pro.webssh.net/assets/44212923/2978bf7e-a276-4b92-9165-3a60069d7091">

I can't give any ETA yet

**Comment by isontheline** [comment]
I think I'm close for a "Designed for iPad" release :
![image](https://github.com/isontheline/pro.webssh.net/assets/44212923/d998d532-3c55-47ff-8b03-2283be8fb8c0)

**Reply by isontheline** [reply]
@tshazar Please subscribe to https://github.com/isontheline/pro.webssh.net/issues/1061

I hope to release by the end of the week
